build.properties

```diff
@@ -1,6 +1,6 @@
 mod_version=1.11.1
-minecraft_version=1.16.5
-forge_version=36.0.14
+minecraft_version=1.16.3
```

Comment:
I would prefer not to lower the MC and Forge compilation target.

It should be possible to keep this unchanged, and make the required changes to also run on lower Forge versions.